scripts/curate_data_inspired.py

```diff
@@ -0,0 +1,390 @@
+        for item in participants_tsv_list:
+            tsv_writer.writerow(item)
+        logger.info(f'participants.tsv created in {path_output}')
+
+
+def create_participants_json(path_output):
+    """
+    Create participants.json file
+    :param path_output: path to the output BIDS folder
+    :return:
+    """
+    # Create participants.json
+    data_json = {
+        "participant_id": {
+            "Description": "Unique Participant ID",
+            "LongName": "Participant ID"
+        },
+        "pathology": {
+            "Description": "Pathology",
+            "LongName": "Pathology name"
+        },
+        "data_id": {
+            "Description": "Subject ID as under duke/mri/",
+            "LongName": "Subject ID"
+        },
+        "institution_id": {
+            "Description": "Institution ID as under duke/mri/",
+            "LongName": "Institution ID"
+        },
+        "institution": {
+            "Description": "Institution ID after conversion to BIDS",
+            "LongName": "BIDS Institution ID"
+        }
+    }
+    with open(os.path.join(path_output, 'participants.json'), 'w') as json_participants:
+        json.dump(data_json, json_participants, indent=4)
+        logger.info(f'participants.json created in {path_output}')
+
+
+def create_dataset_description(path_output):
+    """
+    Create dataset_description.json file
+    :param path_output: path to the output BIDS folder
+    :return:
+    """
+    dataset_description = {"BIDSVersion": "BIDS 1.8.0",
+                           "Name": "inspired"
+                           }
+    with open(os.path.join(path_output, 'dataset_description.json'), 'w') as json_dataset_description:
+        json.dump(dataset_description, json_dataset_description, indent=4)
+        logger.info(f'dataset_description.json created in {path_output}')
+
+
+def create_README(path_output):
+    """
+    Create README file
+    :param path_output: path to the output BIDS folder
+    :return:
+    """
+    with open(os.path.join(path_output, 'README'), 'w') as readme_file:
+        readme_file.write(f'# INSPIRED\n\nThis is an MRI dataset for the INSPIRED project.\n\n## dataset structure\n\n'
+                          f'The dataset contains data from two centers (Toronto, Zurich) across three pathologies ('
+                          f'DCM, SCI, HC). The following images are included:\n\nSpinal cord MRI data:\n- DWI (A-P and '
+                          f'P-A phase encoding)\n- T1w sag\n- T1w sag\n- T1w sag\n- T2star tra\nBrain MRI data:\n- DWI '
+                          f'(A-P and P-A phase encoding)\n- MPM (multi-parameter mapping)\n\n## naming convention\n\n'
+                          f'sub-<site><pathology>XXX\n\nexample:\nsub-zurichDCM001\n\nNote: the label `bp-cspine` is '
+                          f'used to differentiate spine images from brain.')
+        logger.info(f'README created in {path_output}')
```

Comment:
impossble to properly review-- because of formatting-- better approach : just put readme int he pR and i reveiw it---